### PR TITLE
Remove inMemory

### PR DESCRIPTION
**Pull request by diana-macrometa**
Need to update examples to remove inMemory

### CODE REVIEW

File: docs/cep/examples/basic-sw-example.md [hunk 1]
```diff
@@ -5,19 +5,16 @@ title: Basic Stream Worker Example
...
 ```sql
+-- Creates a Macrometa stream to consume events from stream workers.
 CREATE STREAM TemperatureStream (sensorId string, temperature double);
-```
 
-This example creates a sink to publish events from stream workers:
+-- Creates a sink to publish events from stream workers:
 
-```sql
 CREATE SINK TemperatureOnlyStream WITH (type='inMemory', topic='Temperature') (temperature double);
```
Comment (diana-macrometa) [inline docs/cep/examples/basic-sw-example.md]: @StoyanTc, how can we rewrite this?

File: docs/cep/query-guide/functions/index.md [hunk 2]
```diff
@@ -124,16 +124,16 @@ This stream application with the name `TestFunctions` creates a stream named `Fo
 On receiving the event, a query is executed that parses string data types using execution function (math) into required formats.
 After that, the data gets inserted into Kafka `SINK` using Apache Kafka IO function.
 
-```js
-@App:name('TestFunctions') 
+```sql
+    @App:name('TestFunctions') 
 
-CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price string, volume string);
+    CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price string, volume string);
```
Comment (diana-macrometa) [inline docs/cep/query-guide/functions/index.md]: @StoyanTc, how can we rewrite this?

File: docs/cep/examples/basic-sw-example.md (hunk 1 above)
Comment (StoyanTc) [reply docs/cep/examples/basic-sw-example.md]: @diana-macrometa - Let's replace `InMemory` by `stream`?

File: docs/cep/query-guide/functions/index.md (hunk 2 above)
Comment (StoyanTc) [reply docs/cep/query-guide/functions/index.md]: The same here - `type='inMemory'` => `type='stream'`.

**Review by StoyanTc — APPROVED** [review]

**Pull request merged**